pkg/file/joiner/joiner.go

```diff
@@ -31,6 +31,18 @@ func NewSimpleJoiner(getter storage.Getter) file.Joiner {
 }
 
 func (s *simpleJoiner) Size(ctx context.Context, address swarm.Address) (dataSize int64, err error) {
+	toDecrypt := len(address.Bytes()) == (swarm.HashSize + encryption.KeyLength)
```

Comment:
should `toDecrypt` be a parameter?

also, isn't the right side of the `==` actually `EncryptedHashSize`?

in any case, a comment here would be nice

Reply:
toDecrypt need not be a parameter since we can find that out by seeing the length of the address itself. 

Also i added some comments